Microsoft.Toolkit.Uwp.Notifications/Toasts/Compat/ToastNotifierCompat.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if WINDOWS_UWP
```

Comment:
ToastNotifierCompat will exist only on UWP? But the signature change for `public static ToastNotifierCompat CreateToastNotifier()` is for all flavors, no? Something feels weird. Is this class used by all flavors or only by UWP?

Reply:
`#if WINDOWS_UWP` is actually true for all flavors that can access the WinRT (UWP) APIs (so also WPF apps, Windows console apps, etc). I know, kind of confusing... I could refactor all the `#if defs` if desired.

So therefore, ToastNotifierCompat will exist for UWP apps, WPF apps, and Windows console apps (but not on ASP.NET servers). Same as ToastNotificationManagerCompat.

Reply:
Got it! No need, I think its ok the way it is.